contributing/testing-best-practices.md

```diff
@@ -0,0 +1,61 @@
+
+# Testing Best Practices
+
+## Types of test
+
+- Unit tests: In Terraform terminology they refer to tests that [validate a resource schema](https://developer.hashicorp.com/terraform/plugin/framework/handling-data/schemas#unit-testing). That is done automatically [here](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/internal/provider/provider_test.go) for all resources and data sources using Terraform Framework Plugin. Additionally, we have general unit testing for testing a resource or unit without calling the external systems like the Atlas Go SDK.
```

Comment:
```suggestion
- Unit tests: In Terraform terminology they refer to tests that [validate a resource schema](https://developer.hashicorp.com/terraform/plugin/framework/handling-data/schemas#unit-testing). That is done automatically [here](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/internal/provider/provider_test.go) for all resources and data sources using [Terraform Framework Plugin](https://developer.hashicorp.com/terraform/plugin/framework). 
Additionally, we have general unit tests for testing a resource or unit without calling the external systems like the Atlas Go SDK.
```

Reply:
not sure what
>unit without calling the external systems like the Atlas Go SDK.

means

Reply:
rephrased to: without calling external systems like MongoDB Atlas.